homeassistant/util/ssl.py

```diff
@@ -58,6 +59,7 @@ class SSLCipherList(StrEnum):
         "ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:"
         "ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256"
     ),
+    SSLCipherList.INSECURE: "DEFAULT",
```

Comment:
```suggestion
    SSLCipherList.INSECURE: "DEFAULT:@SECLEVEL=0",
```

I think we want everything